tests/framework/installer/ceph_installer.go

```diff
@@ -183,7 +183,56 @@ func (h *CephInstaller) CreateK8sRookCluster(namespace, systemNamespace string,
 		NautilusVersion)
 }
 
-// CreateK8sRookCluster creates rook cluster via kubectl
+// CreateK8sRookClusterWithPVCs creates rook cluster with OSDs over pvc via kubectl
+func (h *CephInstaller) CreateK8sRookClusterWithPVCs(namespace, systemNamespace, storeType string,
```

Comment:
Why do we need this method for installing on PVCs? I thought everything in the tests would be the same except the CephCluster CR spec would be different.

Reply:
removed it